src/server/dask/taskspec.rs

```diff
@@ -0,0 +1,90 @@
+use crate::common::Map;
+use crate::scheduler::TaskId;
+use crate::server::dask::dasktransport::{Frames, SerializedMemory, SerializedTransport};
+use crate::server::dask::key::DaskKey;
+use crate::server::dask::messages::client::{ClientTaskSpec, DirectTaskSpec};
+use serde::{Deserialize, Serialize, Serializer};
+
+#[derive(Serialize, Debug)]
+pub struct DaskTaskSpec<'a> {
+    function: rmpv::Value,
```

Comment:
Cílem je, že Dask pošle Dask specific věci do Gateway a Gateway je přetransformuje na blackbox, který bude striktně v (zatím inline) Msgpacku?

Reply:
Yes exactly. The goal hide task specific configurations into blackbox data, that is transparent for server/workers. Only gateway and subworkers have to understand them.